tests/objc/Example.h

```diff
@@ -20,14 +20,21 @@ struct large {
 };
 
 @interface Example : BaseExample {
+
     int _intField;
     Thing *_thing;
     id<Callback> _callback;
+    int _ambiguous;
 }
 
+#if (XCODE_VERSION >= 80000)
```

Comment:
I think it might be better to use Clang's `__has_extension` macro here to check whether class properties are available. `__has_extension(objc_class_property)` returns `1` if you can use them and `0` if not.